src/modules/nemascan-proxy/publish_container.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+reload_git=''
+tag=''
+
+print_usage() {
+  printf "Usage: %s [-r (reload_git)] -t tag_name" $0;
+}
+
+
+# Get command line arguments
+while getopts 'rt:h' flag; do
+  case "${flag}" in
+    r) reload_git='true' ;;
+    t) tag="${OPTARG}" ;;
+    h) print_usage
+       exit 0 ;;
+    *) print_usage
+       exit 1 ;;
+  esac
+done
+
+
+# Get current environment
+if [ -z ${ENV+x} ]; then
+    echo "Must set var ENV";
+    exit 1;
+else
+    echo "ENV is set to '$ENV'";
+fi
+
+# Map environment to GCP project
+if [ $ENV = "qa" ]; then
+    gcp_project_id="mti-caendr"
+elif [ $ENV = "development" ]; then
+    gcp_project_id="andersen-lab-dev-330218"
+else
+    echo "Invalid ENV '$ENV'"
+    exit 1;
+fi
+echo "Using GCP project ${gcp_project_id}"
+
+# Ensure container tag passed as argument
+if [ -z ${tag} ]; then
+    echo "Must provide value for tag";
+    exit 1;
+else
+    echo "Publishing with tag '${tag}'";
+fi
+
+
+set +a
+source ../../../env/${ENV}/global.env
+
+if [ -n "${reload_git}" ]; then
+    rm -rf ./NemaScan
+fi
+git clone --depth 1 git@github.com:northwestern-mti/NemaScan.git
+
+
+docker build --no-cache -t gcr.io/${gcp_project_id}/nemascan-nxf:${tag} -f NemaScan/Dockerfile ./NemaScan
+docker push gcr.io/${gcp_project_id}/nemascan-nxf:${tag}
```

Comment:
make this line (push) conditional on the previous one (concatenate both with &&), or set a bash flag `-e` to exit if any errors are found.
```bash
set -e
```

Reply:
Added `-e` in a73a67f3ba97a605e26034f645fba96b9453643a